napari/utils/config.py

```diff
@@ -21,27 +22,11 @@ def _set(env_var: str) -> bool:
 
 Async Loading
 -------------
-Image layers will use the ChunkLoader to load data instead of loading
-the data directly. Image layers will not call np.asarray() in the GUI
-thread. The ChunkLoader will call np.asarray() in a worker thread. That
-means any IO or computation done as part of the load will not block the
-GUI thread.
-
-Set NAPARI_ASYNC=1 to turn on async loading with default settings.
+Deprecated.
```

Comment:
I know this was my change, but we might want to add a bit more detail here. E.g. that they should use the setting directly instead.

Reply:
Actually... I reused the same environment variable name to provide consistency with what people expect. I am not 100% clear on how all the settings stuff works, but its my understanding that [this](https://github.com/kcpevey/napari/pull/28/files#diff-a43f22d8d922f45d8904f62c14e401fc31fe943e77f293fe377f5a97deea3597R16) allows it to be set via env var. If that's true then the statement `Set NAPARI_ASYNC=1 to turn on async loading` is also true.

Reply:
Understood. My general point here is that if we're deprecating this and returning some safe, fixed value, then we should explain why it was deprecated (because the corresponding functionality was removed) and what they should use instead.

The other high level option is just to remove these in which case we don't have to worry about these things. You could float that idea with the community - even though this is part of the public API, maybe people will be OK with it?